OMG.Classroom/ApplicationLayer/Services/AssignmentService.cs

```diff
@@ -28,7 +28,7 @@ public async Task DeleteAsync(Guid id)
             await _assignmentRepository.DeleteAsync(id);
         }
 
-        public async Task<List<AssignmentDtoWithId>> ReadAll()
+        public async Task<List<AssignmentDto>> ReadAllAssync()
```

Comment:
1. Typo - not Assync but Async
2. For read and read all - dtos with id => change it everywhere!!!